client/src/components/Workflow/Editor/Index.vue

```diff
@@ -346,6 +347,7 @@ export default {
         },
         onAdd(node) {
             this.nodes[node.id] = node;
+            this.requiresReindex = true;
```

Comment:
This is too aggressive and forces a reload even when you just loaded the workflow

Reply:
Ahhh - I did not test that.

This would have been a great feature to have UI tests for 😅.

Reply:
Certainly looked fine in first glance 😆